### PR TITLE
Update pick documentation

### PR DESCRIPTION
**Pull request by olaaustine**
[ENSVAR-5409](https://www.ebi.ac.uk/panda/jira/browse/ENSVAR-5409)
Merge #https://github.com/Ensembl/ensembl-variation/pull/942 and #https://github.com/Ensembl/ensembl-vep/pull/1336 first

### CODE REVIEW

File: docs/htdocs/info/docs/tools/vep/script/vep_other.html
Comment (ens-ap5) [inline docs/htdocs/info/docs/tools/vep/script/vep_other.html:258]: Please also delete "This is the option", as it makes no sense without the other half of the sentence.

File: docs/htdocs/info/docs/tools/vep/vep_formats.html
Comment (ens-ap5) [inline docs/htdocs/info/docs/tools/vep/vep_formats.html:301]: Is this information only relevant to the www site? If so, please change the protocol to https - if not, make it a relative link.

File: docs/htdocs/info/docs/tools/vep/script/vep_other.html
Comment (ens-ap5) [inline docs/htdocs/info/docs/tools/vep/script/vep_other.html:265]: Is this information only relevant to the www site? If so, please change the protocol to https - if not, make it a relative link.

**Review by ens-ap5 — CHANGES_REQUESTED** [review]
Small tweaks required - can you get them done before the end of this week so we can merge them into the new release?

**Review by nuno-agostinho — APPROVED** [review]
All seems good to me!

File: docs/htdocs/info/docs/tools/vep/vep_formats.html
Comment (ens-ap5) [reply docs/htdocs/info/docs/tools/vep/vep_formats.html:301]: I don't see what's changed here - it should either be "https://www.ensembl.org/info/genome/genebuild/mane.html" or "/info/genome/genebuild/mane.html"

**Review by ens-ap5 — CHANGES_REQUESTED** [review]
Not all requested changes seem to have been implemented.

**Comment by nuno-agostinho** [comment]
Hey @ens-ap5, @olaaustine is currently on leave so I updated the PR with the changes you mentioned. I used relative links for consistency with the rest of the file.

The PRs mentioned in the description were already merged, so you can please merge this PR if you find it fine now. Tell me otherwise.

Thanks! 😊

**Review by ens-ap5 — APPROVED** [review]

**Pull request merged**
